dev_cmds/clean-dev.ts

```diff
@@ -0,0 +1,40 @@
+import { execPiped, runAsyncMain } from "devcmd";
+import { YARN_COMMAND } from "./utils/commands";
+import { devcmdPackageDir, devPath } from "./utils/paths";
+import fs from "fs-extra";
+
+async function main() {
+
+  if(!await fs.pathExists(devPath)) {
+    process.stderr.write('No development folder found!\n')
+    process.exit(1)
+  }
+
+  try {
+    await execPiped({
+      command: YARN_COMMAND,
+      args: ["unlink", "devcmd"],
+      options: {
+        cwd: devPath,
+      },
+    });
+  } catch {}
+
+  try {
+    await execPiped({
+      command: YARN_COMMAND,
+      args: ["unlink"],
+      options: {
+        cwd: devcmdPackageDir,
+      },
+    });
+  } catch {}
+
+  try {
+    await fs.remove(
+      devPath
+    );
+  } catch {}
```

Comment:
I think any failures here should be logged (e.g. as warnings), though the script may continue.
the user should be notified if the clean up didn't work :)